cgmanifest.json

```diff
@@ -26839,6 +26859,16 @@
         }
       }
     },
+    {
```

Comment:
remove

Reply:
close and reopen the PR seemed to fix whatever was going on.  cargo2rpm and rust-packaging should be gone now.